www/main/common/store/user-settings/get-saved-user-settings.js

```diff
@@ -3,14 +3,17 @@ import convertObjToCamelCase from '../../utils/convert-object-to-camel-case';
 const electron = require('electron');
 const fs = require('fs');
 const path = require('path');
+
 const { settings } = require('../../../../configs/user-settings.json');
 
 let userDataPath;
 
 if (electron.app) {
   userDataPath = electron.app.getPath('userData');
 } else {
-  const app = require('@electron/remote').app;
+  /* eslint-disable global-require */
+  const { app } = require('@electron/remote');
```

Comment:
This exception is used at multiple places through out the code. Linting rules want require to be the first thing in code, but we are using require in conditionals or with some code logic like above.